### PR TITLE
12 create faq section of the homepage

### LINKED ISSUE #12: Create : FAQ Section of the Homepage
## Summary

Create some Frequently Asked Questions and their answers. Usage of an illustration/image is highly encouraged.

Design and responsiveness must be readied with Tailwind, snapshot/unit tests should also be written.

### Screenshots

Ekran görüntüsü 2022-10-21 190311

### Basic example

N/A

### Motivation

To clarify the user's possible questions before they actually start using the webpage.

### PR DESCRIPTION
**Pull request by MARWAHAKMI**
FAQ section done with tests

### CODE REVIEW

File: src/components/FAQ/faq.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "questions": [{
+    "question": "What is the return policy?",
+    "Answer": "There must be a deal between the customers and the sellers to determine how returning items should be and our staff is not responsible of it"
+
+  }, {
+    "question": "Can I cancel my order",
```
Comment (sarahaboudalal) [inline src/components/FAQ/faq.json]: me being a perfectionist: you forgot to add a '?'

**Review by sarahaboudalal — COMMENTED** [review]
I added a boring comment.

**Review by ayubabdullah — APPROVED** [review]
Great work guys, well done

I has 2 comments 

1- it will be a better user experience when a user clicks on one question to open another close automatically

2- Having a transition between opening and closing will be great

**Comment by MARWAHAKMI** [comment]
> Great work guys, well done
> 
> I has 2 comments
> 
> 1- it will be a better user experience when a user clicks on one question to open another close automatically
> 
> 2- Having a transition between opening and closing will be great

Thank you, Ayub, for the auto-close Accordion: I planned to do that but then cancelled that. You changed my mind. I will fix it.
For the transition, there’s already a transition on opening the accordion.

**Comment by OmarBassatni97** [comment]
I checked the code and I see a transition but when you open the website you don't feel the transition can you check please and shouldn't the arrow be downwards before opening the question and upwards when its open?

**Comment by ayubabdullah** [comment]
> on

Exactly as omer said you don't feel the transition on opening and closing my be there is a problem with transition time

File: src/components/FAQ/faq.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+     
+  }, {
+    "question": "How do I track my order?",
+    "Answer":"You can have the phone number of the company which does the delivering and ask them where your order is"}
+    ,{
+      "question": "What is the policy for late/non-delivery of items ordered online?",
+      "Answer": "Our customer service team will be there alwyas to resolve your issue,you can contact us"
+      
+    } ]
+}
```
Comment (Iseluin) [inline src/components/FAQ/faq.json]: Please add the missing punctuation marks like "?" and "." . And please change the "Answer" key to lowercase.

**Review by Iseluin — CHANGES_REQUESTED** [review]
Great work people. I left a small comment on the punctuation in the .json file and I agree with other reviews from sarah, ayub and omer. Also, when you open and close the disclosures, the component grows and shrinks. We dont want that to happen, please try to fix that by giving the component a max height maybe ? Everything else looks really great!

**Review by sarahaboudalal — CHANGES_REQUESTED** [review]
Heyy Mariam, I was looking into ur code. It looks great but I agree with omar and ayub and I have some other notes:
1. I think it is conventional to make the arrow point down when the question section is closed and up when it's opened
2. the last arrow is kinda smaller and less bold
3. It's also conventional to not allow more than one question to be opened so that the component doesn't take up much space.
4. lastly, pull the changes from development branch into urs

**Comment by MARWAHAKMI** [comment]
Thank you, Sarah, for the notes; I will work on them.
btw it's Marwa and Mawa, not Mariam

**Comment by sarahaboudalal** [comment]
> Thank you, Sarah, for the notes; I will work on them. btw it's Marwa and Mawa, not Mariam

lol I'm dead...

**Review by sarahaboudalal — APPROVED** [review]
It looks gorgeous Marwa! Well done🤩

File: src/components/FAQ/index.jsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
+import { Disclosure, Transition } from '@headlessui/react';
```
Comment (sarahaboudalal) [inline src/components/FAQ/index.jsx:1]: Perhaps changing the file name to 'FAQ.jsx'

File: src/components/FAQ/index.jsx [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+                      className="px-4 py-4 text-md text-gray-500 text-left"
+                    >
+                      {item.answer}
+                    </Disclosure.Panel>
+                  </Transition>
+                </>
+              )}
+            </Disclosure>
+          ))}
+        </div>
```
Comment (sarahaboudalal) [inline src/components/FAQ/index.jsx:83]: Idk why but when I click again on the opened section it doesn't close unless I opened another question, can we fix that?

**Review by sarahaboudalal — COMMENTED** [review]
I suggested some changes in the comments

File: src/components/FAQ/index.jsx (hunk 4 above)
Comment (Iseluin) [reply src/components/FAQ/index.jsx:83]: @MARWAHAKMI @mawaidris I agree with this.

File: src/components/FAQ/index.jsx (hunk 3 above)
Comment (Iseluin) [reply src/components/FAQ/index.jsx:1]: @sarahaboudalal lets keep those minimal things to the bug bashing part.

File: src/components/FAQ/index.jsx (hunk 4 above)
Comment (MARWAHAKMI) [reply src/components/FAQ/index.jsx:83]: @Iseluin please can you check it

Comment (Iseluin) [reply src/components/FAQ/index.jsx:83]: @MARWAHAKMI beautiful. Are you gonna change anything else or are you ok with merging this ?

**Review by Iseluin — APPROVED** [review]
👍🏻 feel free to squash and merge if you dont want no more improvements.

File: src/components/FAQ/index.jsx (hunk 4 above)
Comment (MARWAHAKMI) [reply src/components/FAQ/index.jsx:83]: I don't think there are any improvements; I will merge pr

**Pull request merged**